utils/containerd/Makefile

```diff
@@ -53,8 +42,8 @@ MAKE_FLAGS += \
 	VERSION=$(PKG_VERSION) \
 	REVISION=$(PKG_SOURCE_VERSION)
 
-ifeq ($(CONFIG_CONTAINERD_SECCOMP),y)
-MAKE_FLAGS += BUILDTAGS='seccomp'
```

Comment:
Here you remove the check for the no longer existing `CONFIG_CONTAINERD_SECCOMP`. But where is the new check, eg. for `CONFIG_KERNEL_SECCOMP_FILTER`?

Reply:
[It seems that tag isn't useful anymore](https://github.com/containerd/containerd/blob/v1.4.1/releases/v1.4.0.toml#L62).
however, I'll remove `libseccomp` as I seemed to have missed it.